ios/FluentUI/Notification/NotificationModifiers.swift

```diff
@@ -8,65 +8,28 @@ import SwiftUI
-                             message: String? = nil,
-                             attributedMessage: NSAttributedString? = nil,
+    func presentNotification(isPresented: Binding<Bool>,
                              isBlocking: Bool = true,
-                             isPresented: Binding<Bool>,
-                             title: String? = nil,
-                             attributedTitle: NSAttributedString? = nil,
-                             image: UIImage? = nil,
-                             trailingImage: UIImage? = nil,
-                             trailingImageAccessibilityLabel: String? = nil,
-                             actionButtonTitle: String? = nil,
-                             actionButtonAction: (() -> Void)? = nil,
-                             showDefaultDismissActionButton: Bool? = nil,
-                             messageButtonAction: (() -> Void)? = nil,
-                             showFromBottom: Bool = true,
-                             overrideTokens: NotificationTokens? = nil) -> some View {
+                             @ViewBuilder notification: @escaping () -> FluentNotification) -> some View {
         self.presentingView(isPresented: isPresented,
                             isBlocking: isBlocking) {
-            FluentNotification(style: style,
-                               isFlexibleWidthToast: isFlexibleWidthToast,
-                               message: message,
-                               attributedMessage: attributedMessage,
-                               isPresented: isPresented,
-                               title: title,
-                               attributedTitle: attributedTitle,
-                               image: image,
-                               trailingImage: trailingImage,
-                               trailingImageAccessibilityLabel: trailingImageAccessibilityLabel,
-                               actionButtonTitle: actionButtonTitle,
-                               actionButtonAction: actionButtonAction,
-                               showDefaultDismissActionButton: showDefaultDismissActionButton,
-                               messageButtonAction: messageButtonAction,
-                               showFromBottom: showFromBottom)
-            .overrideTokens(overrideTokens)
+            notification()
         }
     }
 }
 
 public extension FluentNotification {
-    /// Provides a custom design token set to be used when drawing this control.
+    /// Specifies a custom background 
+    func backgroundGradient(_ gradientInfo: GradientInfo?) -> FluentNotification {
+        state.backgroundGradient = gradientInfo
+        return self
+    }
+
+    /// An optional gradient to use as the background of the notification.
+    ///
+    /// If this property is nil, then this notification will use the background color defined by its design tokens.
     func overrideTokens(_ tokens: NotificationTokens?) -> FluentNotification {
```

Comment:
The comments on both modifiers are not quite right.

Reply:
Copy-paste failure.